qemu/tests/multicast_iperf.py

```diff
@@ -68,6 +68,12 @@ def server_start(cmd, catch_data):
         exit_status = process.system(app_check_cmd, ignore_status=True,
                                      shell=True)
 
+        # Install gcc-c++ to compile iperf
+        gcc_cpp_ver_chk_cmd = params.get("gcc_cpp_ver_chk_cmd")
+        gcc_cpp_ver = process.system_output(gcc_cpp_ver_chk_cmd, shell=True)
```

Comment:
You don't need the `ver` in these variable, since the version (if `ver` stands for version) it not required in this case.

Reply:
Updated.